src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -25,6 +26,8 @@ class DacFxService
         private static readonly Lazy<DacFxService> instance = new Lazy<DacFxService>(() => new DacFxService());
         private readonly Lazy<ConcurrentDictionary<string, DacFxOperation>> operations =
             new Lazy<ConcurrentDictionary<string, DacFxOperation>>(() => new ConcurrentDictionary<string, DacFxOperation>());
+        private Lazy<ConcurrentDictionary<string, TSqlModel>> modelResults =
```

Comment:
`projectModels` would be a better name IMO

And please add a comment explaining what the key value is (the project URI in this case)